Project.toml

```diff
@@ -20,6 +20,7 @@ RandomExtensions = "fb686558-2515-59ef-acaa-46db3789a887"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 Singular = "bcd08a7b-43d2-5ff7-b6d4-c458787f915c"
 TOPCOM_jll = "36f60fef-b880-50dc-9289-4aaecee93cc3"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
```suggestion
```
What is this needed for? For the tests to succeed, it should be enough for it to be contained in `test/Project.toml`.